benchmarking/smartseq2/merge_picard_metrics/merge_picard_metrics.wdl

```diff
@@ -0,0 +1,64 @@
+task MergePicardMet {
+  String target_dir
+  String met_name
+  String uuid
+  String output_name
+  command {
+    set -e
+    gsutil cp  ${target_dir}/merge_picard_mets.py ./   
+    python merge_picard_mets.py  -u ${uuid} -m ${met_name} -o ${output_name}
+  }
+
+  output {
+    File merged_metrics = "${output_name}"
+  }
+  runtime {
+    docker:"gcr.io/broad-dsde-mint-dev/analysis-tools:0.0.3"
+    memory:"3.75 GB"
+    disks: "local-disk 50 HDD"
+  }
+}
+task AppendMetricsFiles { 
+  Array[File] met_files
+  String output_name
+  command {
+    set -e
+    isheader=0
+    for f in ${sep=' ' met_files}; do
+      if [[ $isheader == 0 ]];then
```

Comment:
Does this only grab the header from the first metrics file and metrics from the other files?

Reply:
Yes. grab header and metrics table from the first file then only grab metrics table from the rest metrics files.

Reply:
right...i blame my eyes for not being to read that correctly :D